composer.json

```diff
@@ -9,13 +9,15 @@
             "EventSauce\\MessageOutbox\\DoctrineOutbox\\": "src/DoctrineOutbox",
             "EventSauce\\MessageRepository\\IlluminateMessageRepository\\": "src/IlluminateMessageRepository",
             "EventSauce\\MessageRepository\\DoctrineMessageRepository\\": "src/DoctrineMessageRepository",
-            "EventSauce\\MessageRepository\\TestTooling\\": "src/MessageRepositoryTestTooling"
+            "EventSauce\\MessageRepository\\TestTooling\\": "src/MessageRepositoryTestTooling",
+            "EventSauce\\MessageRepository\\DynamoDBMessageRepository\\": "src/DynamoDBMessageRepository"
         }
     },
     "require": {
         "eventsauce/eventsauce": "^1.0",
         "eventsauce/backoff": "^1.0",
-        "doctrine/dbal": "^3.1"
+        "doctrine/dbal": "^3.1",
+        "aws/aws-sdk-php": "^3.0"
```

Comment:
Did you considered https://async-aws.com/clients/dynamodb.html ?
Lighter , smaller memory/cpu footprint, better DX.

/cc @Nyholm

Reply:
Thanks for the suggestion @jderusse. I've used `async-aws` before and the experience was great.

Open to change It if @frankdejonge agrees.

Reply:
All good for me 👍

Reply:
done